gremlin-go/driver/graphBinary.go

```diff
@@ -621,6 +624,7 @@ func vertexPropertyReader(buffer *bytes.Buffer, typeSerializer *graphBinaryTypeS
 }
 
 //Format: {Labels}{Objects}
+// TODO: Path serialization is currently incomplete as Labels are represented as list of lists due to lack of native set types in go. Fully functional Path serialization will be implemented when set is implemented in AN-1032
```

Comment:
I thought this was fixed with our path type? Are our paths incorrect?

Reply:
Looks like a mix-up from my merge attempt. Will delete.